tests/end2end/features/navigate.feature

```diff
@@ -65,3 +65,8 @@ Feature: Using :navigate
         And I open data/numbers/1.txt
         And I run :navigate increment
         Then the error "No number found in URL!" should be shown
+        
+    Scenario: navigate matching for multiline links
+        when I open /data/navigate/multilinelinks.html
```

Comment:
This should be `When` (with a capital `w`), and you don't need the leading `/` before `data` (not sure if it works with it, but in all other places it's used without)